css/index.css

```diff
@@ -19,262 +19,145 @@
-}
-.main-actions{
-    display: block;
-    margin: 0 auto;
-    align-items: center;
-}
-.main-actions > .btn:not(:last-child){
-    margin-right: 1.5rem;
+.feature {
+  max-width: 33%;
+  padding: 0 1.8rem;
 }
 
-@media screen and (max-width: 900px) {
-    .main-paragraph{
-        max-height: 90%;
-        max-width: 75%;
-        font-size: 16px;
-        padding: 0 0.5rem;
-        margin-bottom: 2rem;
-    }
-    .main-actions > .btn:not(:last-child){
-        margin-bottom: 1rem;
-        margin-right: 0;
-    }
-    .main-actions > .btn{
-        display: block;
-    }
+.feature-title {
+  color: #28afce;
+  text-align: center;
+  margin-bottom: 1rem;
 }
 
-/* Buttons */
-
-.btn{
-    padding: 1rem 2rem ;
-    border-radius: 40% /100%;
-    font-size: 17px;
-    white-space: nowrap;
-    color: black;
-}
-.btn:hover{
-    cursor: pointer;
-}
-.btn-outline--primary{
-    background: transparent;
-    border: 2px #28afce solid ;
-    padding: 0.9rem 2rem ;
-}
-.btn-outline--secondary{
-    background: transparent;
-    border: 2px #466e71 solid;
-    padding: 0.9rem 2rem ;
-}
-.btn--primary{
-    background-color: #28afce;
-    color: #fff !important;
-}
-.btn--secondary{
-    background-color: #466e71 ;
-    color: #fff !important;
+.feature-paragraph {
+  text-align: center;
 }
 
-a.btn{
-    text-decoration: none;
-}
-a.btn:visited{
-    color:inherit;
-}
-
-/* Features sections */
+/* Footer */
 
-#features-section{
-    height: auto;
-    background-color: #f6f6f6;
-    padding:5rem 0;
-}
-.features-box{
-    max-width: 900px;
-    margin: 0 auto;
-    display: flex;
-    justify-content: start;
-}
-.feature{
-    max-width: 33%;
-    display: inline-block;
-    padding: 0 1.8rem;
-}
-.feature-title{
-    color: #28afce;
-    font-weight: 600;
-    white-space: nowrap;
-    text-align: center;
-    margin-bottom: 1rem;
+footer {
+  background-color: #155195;
+  padding: 3rem;
+  color: #f6f6f6;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
-.feature-paragraph{
-    color: black;
 
-    display: block;
-    text-align: center;
-    font-size: medium;
-    font-weight: 300;
-}
+/* Mobile Responsive */
 
 @media screen and (max-width: 900px) {
-    .features-box{
-        flex-direction: column;
-        align-items: center;
-    }
-    .feature{
-        display: block;
-        margin-bottom: 1rem;
-        max-width: 70%;
-    }
-
-}
-
-/* Footer */
-
-footer{
-    background-color: #155195;
-    padding: 3rem;
-    color: #f6f6f6;
-    display: flex;
+  .main-paragraph {
+    max-height: 90%;
+    max-width: 75%;
+    font-size: 16px;
+    padding: 0 0.5rem;
+    margin-bottom: 2rem;
+  }
+  .main-actions > .btn:not(:last-child) {
+    margin-bottom: 1rem;
+    margin-right: 0;
+  }
+  .main-actions > .btn {
+    display: block;
+  }
+  .features-box {
     flex-direction: column;
     align-items: center;
+  }
+  .feature {
+    display: block;
+    margin-bottom: 1rem;
+    max-width: 70%;
+  }
 }
```

Comment:
what about changing the width of the logo here?` .main-logo`

Reply:
Yes sure